src/Appwrite/Utopia/Response/Model/Func.php

```diff
@@ -87,6 +99,54 @@ public function __construct()
                 'default' => 15,
                 'example' => 1592981237,
             ])
+            ->addRule('entrypoint', [
+                'type' => self::TYPE_STRING,
+                'description' => 'The entrypoint file used to execute the deployment.',
+                'default' => '',
+                'example' => 'index.js',
+            ])
+            ->addRule('buildCommand', [
+                'type' => self::TYPE_STRING,
+                'description' => 'The build command used to build the deployment.',
+                'default' => '',
+                'example' => 'npm run build',
+            ])
+            ->addRule('installCommand', [
+                'type' => self::TYPE_STRING,
+                'description' => 'The install command used to build the deployment.',
+                'default' => '',
+                'example' => 'npm install',
```

Comment:
Should be changed to a single commands array.

Reply:
Updated response model, logic, and params on endpoints